### ISSUE TITLE
Weird transparency with stacked tabs

### ISSUE THREAD

**Issue by EaZyStudiio** [samuelsong70/obsidian-dynamic-background#3]
![Obsidian_Ze77lRQQN2](https://user-images.githubusercontent.com/55167211/204247428-02625473-9ba8-4a96-aa23-05a054aa2599.png)
Green is the previos tab and red is the new tab which is stacked but I can still see the previous tab

Amazing Plugin by the way

**Comment by samuelsong70** [comment]
Thanks for your reminder. I haven't used the stacking method before, and I tried it just now, and there is indeed the problem you mentioned. I try to find the problem and fix it as soon as possible.

**Comment by EaZyStudiio** [comment]
Thank you. Loveing my time with obsidian mostly due to the rain in the background 🤣

**Comment by samuelsong70** [comment]
I analyzed your feedback and found the reason. The reason for the problem is that Obsidian did not correctly set the is-hidden attribute of the tab in stack mode, so the content of the tab that should be hidden is displayed. This is no problem for other plugins, but it will cause problems for Obsidian Dynamic Background, because to display the background image, we changed the background color of the editor to transparent. I've reported the issue to Obsidian and hope they can address it as soon as possible.
